src/utils/compare-version.ts

```diff
@@ -0,0 +1,15 @@
+export function compareVersion(versionA: string, versionB: string) {
+  const A = versionA.split('.');
+  const B = versionB.split('.');
+  for (let i = 0; i < Math.min(A.length, B.length); i++) {
+    if (A[i] !== B[i]) {
+      return parseInt(A[i]) > parseInt(B[i]) ? 1 : -1;
+    }
+  }
+  if (A.length === B.length) {
+    return 0;
+  } else if (A.length > B.length) {
+    return 1;
+  }
+  return -1;
```

Comment:
Probably don't need to return `-1`, `0` and `1` as explicitly.

I think `return parseInt(A[i]) - parseInt(B[i])` and `return A.length - B.length` should work just as well.